### PR TITLE
[MIG] web_select_all_companies: Backporting from 15.0 to 14.0

### PR DESCRIPTION
**Pull request by ajaniszewska-dev**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]
could you check pre-commit?

**Comment by twalter-c2c** [comment]
@ivantodorovich pre-commit fixed

File: web_select_all_companies/static/src/xml/switch_all_company_menu.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                        <t t-if="widget.isAllCompaniesSelected">
+                            <i class="fa fa-fw fa-check-square pt-2" />
+                        </t>
+                        <t t-else="">
+                            <i class="fa fa-fw fa-square-o pt-2" />
+                        </t>
+                    </span>
+                </div>
+                <div
+                    class="d-flex flex-grow-1 align-items-center py-0 log_into pl-3 o_py"
```
Comment (florentx) [inline web_select_all_companies/static/src/xml/switch_all_company_menu.xml]: Changing this class is an improvement:

```suggestion
                    class="d-flex flex-grow-1 align-items-center py-0 toggle_company pl-3 o_py"
```

Before:
- user has to click the checkbox to toggle all companies
- when user click the text "Select all companies", behavior is confusing

After:
- user can click either the checkbox or the text to select all companies.

**Review by florentx — COMMENTED** [review]
Suggestion for an improvement, after feedback from a user.

File: web_select_all_companies/static/src/xml/switch_all_company_menu.xml (hunk 1 above)
Comment (ajaniszewska-dev) [reply web_select_all_companies/static/src/xml/switch_all_company_menu.xml]: @florentx  indeed it was causing issue, updated PR - thank you

**Review by florentx — APPROVED** [review]
LG, and works fine

**Pull request merged**